src/components/Map.js

```diff
@@ -53,13 +54,14 @@ const Map = ( props ) => {
       </BaseMap>
     </div>
   );
-};
+});
 
 Map.propTypes = {
   children: PropTypes.node,
   className: PropTypes.string,
   defaultBaseMap: PropTypes.string,
   mapEffect: PropTypes.func,
+  ref: PropTypes.shape({ current: any }),
```

Comment:
can you update `any` to `PropTypes.any` and remove the import so it conforms with the rest of the proptype statements?